examples/twitter-compose-with-typeahead/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Twitter compose box with typeahead",
```

Comment:
Google recommends that `short_name` is [12 characters maximum](https://developer.chrome.com/docs/extensions/mv2/manifest/name/#short_name).

Reply:
I've managed 16 (see dc606664).